src/components/filesystem/index.js

```diff
@@ -561,9 +566,13 @@ module.exports = {
 	    for (var i=0; i < this.selectedFiles.length; i++) {
             var file = this.selectedFiles[i];
             console.log("deleting: " + file.getFileProperties().name);
+            this.showSpinner = true;
             var that = this;
             file.remove(this.getContext().network, this.currentDir)
-                .thenApply(function(b){that.currentDirChanged()});
+                .thenApply(function(b){
+                        that.currentDirChanged();
+                        that.showSpinner = false;
```

Comment:
this will close the spinner after the first of the files is removed. Need to have a countdown.